templates/gitlab.rb.j2

```diff
@@ -53,6 +53,59 @@ gitlab_rails['ldap_servers'] = {
 }
 {% endif %}
 
+{% if gitlab_omniauth_enabled is defined %}
+gitlab_rails['omniauth_enabled'] = "{{ gitlab_omniauth_enabled }}"
+{% endif %}
+{% if gitlab_omniauth_allow_bypass_two_factor is defined %}
```

Comment:
Not a real problem, but to be more readable, I like to indent Jinja conditions:
```suggestion
{% if gitlab_omniauth_allow_bypass_two_factor is defined %}
{%   if gitlab_omniauth_allow_bypass_two_factor is string or gitlab_omniauth_allow_bypass_two_factor is boolean %}
gitlab_rails['omniauth_allow_bypass_two_factor'] = "{{ gitlab_omniauth_allow_bypass_two_factor }}"
{%   else %}
gitlab_rails['omniauth_allow_bypass_two_factor'] = [ "{{ gitlab_omniauth_allow_bypass_two_factor|join(', ') }}" ]
{%   endif %}
{% endif %}
```

Reply:
Not mandatory to merge

Reply:
I did reindent all code to comply